docs/topics/mta-rn-known-issues-7-0-0.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-0_{context}"]
+= Known issues
```

Comment:
Ensure you speak to Dylan, Ilanit, and Shveta about which issues must be included